apps/server/src/config/database.config.ts

```diff
@@ -1,7 +1,7 @@
-import { MysqlConnectionOptions } from 'typeorm/driver/mysql/MysqlConnectionOptions';
+import { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions';
 
-export default (): MysqlConnectionOptions => ({
-  type: 'mysql',
+export default (): PostgresConnectionOptions => ({
+  type: 'postgres',
   host: process.env.DATABASE_HOST,
   port: parseInt(process.env.DATABASE_PORT!) ?? 3306,
```

Comment:
`port: Number(process.env.DATABASE_PORT) ?? 3306` would be better

Reply:
updated to use `Number` instead of `parseInt`